monai/bundle/properties.py

```diff
@@ -159,6 +159,21 @@
         BundleProperty.REQUIRED: True,
         BundlePropertyConfig.ID: "device",
     },
+    "dataset_dir": {
```

Comment:
Hi @ericspod ,

Just want to explain the change: I added these properties for the inference part to be consistent with the train part. These properties can be useful to override the inference workflow with user's own data.

Thanks.